packages/maskbook/src/plugins/RedPacket/constants.ts

```diff
@@ -3,10 +3,12 @@
  * !! Please make sure it also be able to recognize the old key.
  */
 export const RedPacketMetaKey = 'com.maskbook.red_packet:1'
+export const RedPacketNftMetaKey = 'com.maskbook.red_packet_nft:1'
 /**
  * !! This ID is used to identify the stored plugin data. Change it will cause data lost.
  */
 export const RedPacketPluginID = 'com.maskbook.red_packet'
+export const RedPacketNftPluginID = 'com.maskbook.red_packet_nft'
```

Comment:
Please do not create multiple plugin IDs. Each plugin should only have 1 plugin ID.